Classes/helpers/class.tx_caretaker_LatestVersionsHelper.php

```diff
@@ -75,10 +77,40 @@ public static function updateLatestTypo3VersionRegistry()
             if (is_array($details) && !empty($details['stable'])) {
                 $stable[$major] = $details['stable'];
             }
+            if (is_array($details) && is_array($details['releases'])) {
+			    $found = false;
```

Comment:
This variable is not necessary. Just set $security[$major] to the latest before the loop.